iphone/Classes/TiUIWebView.m

```diff
@@ -83,7 +83,9 @@ - (WKWebView *)webView
     _willHandleTouches = [TiUtils boolValue:[[self proxy] valueForKey:@"willHandleTouches"] def:YES];
 
     _webView = [[WKWebView alloc] initWithFrame:[self bounds] configuration:config];
-
+#if TARGET_OS_SIMULATOR && __IPHONE_OS_VERSION_MAX_ALLOWED >= 160400
+    _webView.inspectable = YES;
```

Comment:
This one is still missing the `[TiUtils isiOSVersionOrGreater:@"16.4"]` check, which is used as a runtime check. The other ones ensure that you can build the app with Xcode < 14.3 (pre-compile statement) :-)